content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+
+| Node Condition    | Eviction Signal                                                                       | Description                                                                                                                  |
+|-------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| `MemoryPressure`  | `memory.available`                                                                    | Available memory on the node has satisfied an eviction threshold                                                             |
+| `DiskPressure`    | `nodefs.available`, `nodefs.inodesFree`, `imagefs.available`, or `imagefs.inodesFree` | Available disk space and inodes on either the node's root filesystem or image filesystem has satisfied an eviction threshold |
+| `PIDPressure`     | `pid.available`                                                                       | Available processes identifiers on the (Linux) node has fallen below an eviction threshold                                   |
+
+The kubelet updates the node conditions based on the configured 
+`--node-status-update-frequency`, which defaults to `10s`.
+
+### Node condition oscillation
+
+In some cases, nodes oscillate above and below soft eviction thresholds without
+holding for the defined grace periods. This causes the reported node condition
+to constantly switch between `true` and `false`, leading to bad eviction decisions.
+
+To protect against oscillation, you can use the `eviction-pressure-transition-period`
+flag, which controls how long the kubelet must wait before transitioning a node
+condition to a different state.
+
+## Reclaiming node level resources {#reclaim-node-resources}
+
+The kubelet tries to reclaim node-level resources before it evicts end-user pods.
+
+When a `DiskPressure` node condition is reported, the kubelet reclaims node-level
+resources based on the filesystems on the node. 
+
+### With `imagefs`
+
+If the node has a dedicated `imagefs` filesystem for container runtimes to use,
+the kubelet does the following:
+
+  * If the `nodefs` filesystem meets the eviction threshlds, the kubelet deletes
+    dead pods and containers. 
+  * If the `imagefs` filesystem meets the eviction thresholds, the kubelet
+    deletes all unused images. 
+
+### Without `imagefs`
+
+If the node only has a `nodefs` filesystem that meets eviction thresholds,
+the kubelet frees up disk space in the following order:
+
+1. Delete dead pods and containers
+1. Delete unused images
+
+## Evicting end-user pods
+
+If the kubelet's attempts to reclaim node-level resources don't bring the eviction
+signal below the threshold, the kubelet begins to evict end-user pods. 
+
+The kubelet uses the following parameters to determine pod eviction order:
+
+1. Pod resource usage relative to defined requests
```

Comment:
This reads as being the same as number 3.  Suggestion: "If pod resource usage exceeds defined requests or not"

Reply:
Out of curiosity, what's the difference between 1 and 3?

Reply:
1. Requests < Usage ?  True or false
2. ...
3. Usage - Requests

Reply:
OH. Okay, point 3 should be clearer - by how much does usage exceed requests?